main.js

```diff
@@ -223,17 +246,24 @@ export class IdDecoder {
    *   with leading non-zero data will error.
    * @param {boolean} [options.multibase=true] - Use multibase encoding to
    *   detect the id format.
-   *
+   * @param {boolean} [options.multihash=false] - Use multihash encoding to
+   *   detect the id format.
+   * @param {number} [options.expectedSize=32] - Optional expected identifier
+   *   size in bytes (only for multihash encoding).
```

Comment:
Should document how to disable:
```suggestion
   * @param {number} [options.expectedSize=32] - Optional expected identifier
   *   size in bytes (only for multihash encoding). Zero to disable check.
```

Reply:
Added 075e0a3